src/Umbraco.Web/Search/IInternalSearchConstants.cs

```diff
@@ -0,0 +1,13 @@
+using System.Collections.Generic;
+
+namespace Umbraco.Web.Search
+{
+    public interface IInternalSearchConstants
```

Comment:
I think this could have a friendlier name. Since this is explicit to the UmbracoTreeSearcher this could be something like `UmbracoTreeSearcherFields`

Reply:
Actually I was trying to find the best name, but I am open for the new name :), Actually good idea with that proposition.